models/keyword.go

```diff
@@ -58,6 +58,14 @@ func FindKeywordById(id int64) (keyword *Keyword, err error) {
 	return keyword, nil
 }
 
+func (k *Keyword) IsBelongTo(u *User) bool {
```

Comment:
First, the naming is a bit awkward. Having two consecutive verbs sounds odd 😅 Apart from the naming, couldn't this be achieved by having a generic method `FindKeywordBy` and then have a list of params to filter. If the query returns nil or an error then the controller can return not found. So that could cover both `FindKeywordById` and `IsBelongTo`.

Reply:
Introduced `FindKeywordBy` functions to cover  both `FindKeywordById` and `IsBelongTo`